scripts/convert_stable_cascade.py

```diff
@@ -121,7 +126,9 @@
     scheduler=scheduler,
     feature_extractor=feature_extractor,
 )
-prior_pipeline.save_pretrained(f"{args.save_org}/StableCascade-prior", push_to_hub=args.push_to_hub)
+prior_pipeline.to(dtype).save_pretrained(
+    f"{args.save_org}/StableCascade-prior", push_to_hub=args.push_to_hub, variant=args.variant
```

Comment:
For FP32, this should be None. This is why https://github.com/huggingface/diffusers/pull/7271/files#r1519574862